.travis.yml

```diff
@@ -0,0 +1,14 @@
+---
+services: docker
+
+env:
+  - distro: centos7
```

Comment:
Looks like it is trying to test centos? https://travis-ci.org/Islandora-Devops/ansible-role-blazegraph/jobs/338288091#L443

Reply:
good call. missed that.